pkg/antctl/raw/helper.go

```diff
@@ -220,3 +225,26 @@ func CreateControllerClientCfg(
 	cfg.Host = fmt.Sprintf("https://%s", net.JoinHostPort(nodeIP, fmt.Sprint(controllerInfo.APIPort)))
 	return cfg, nil
 }
+
+type PodFileCopy interface {
+	CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error
+}
+
+type PodFile struct {
+	RestConfig *rest.Config
+	Client     kubernetes.Interface
+}
+
+func (p *PodFile) CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error {
+	dir, fileName := filepath.Split(srcPath)
+	cmd := fmt.Sprintf("cd %s; tar cf - %s", dir, fileName)
```

Comment:
instead of `cd`, could we use the `-C` option for `tar`?

For example:
```go
cmd := ["tar"]
if dir != "" {
        cmd = append(cmd, "-C", dir) 
}
cmd = append(cmd, "cf", "-", fileName)
```

This way, you should be able to skip the shell wrapping and exec the `tar` program directly. It's always better to do it this way and avoid the shell dependency in the container.